config/android/BUILD.gn

```diff
@@ -55,14 +55,18 @@ config("compiler") {
   if (is_clang) {
     if (current_cpu == "arm") {
       abi_target = "arm-linux-androideabi"
+      cflags += [ "-D__ANDROID_API__=16" ]
     } else if (current_cpu == "x86") {
       abi_target = "i686-linux-androideabi"
+      cflags += [ "-D__ANDROID_API__=16" ]
     } else if (current_cpu == "arm64") {
       abi_target = "aarch64-linux-android"
+      cflags += [ "-D__ANDROID_API__=21" ]
     } else if (current_cpu == "x64") {
       # Place holder for x64 support, not tested.
       # TODO: Enable clang support for Android x64. http://crbug.com/539781
       abi_target = "x86_64-linux-androideabi"
+      cflags += [ "-D__ANDROID_API__=21" ]
```

Comment:
after the __for the Unified Headers__ changes, we need set `-D__ANDROID_API__ ` parameter